utils/init.js

```diff
@@ -4,9 +4,13 @@ const pkgJSON = require('./../package.json');
 const updateNotifier = require('update-notifier');
 const unhandledError = require('cli-handle-unhandled');
 
-module.exports = async () => {
+const MINIMUM_NODE_JS_VERSION = '10';
+
+module.exports = async dangerous => {
 	unhandledError();
-	checkNode(`10`);
+	if (!dangerous) {
```

Comment:
You didn't handle the case when `dangerous-version` flag is set.

Reply:
@ahmadawais : By default, dangerous is false. I have added the logic in such a way that `checkNode` is called only if `dangerous` flag is not set. If `dangerous` is set, it will not go inside the if block at all. Does it make sense?